lib/RoadizRozierBundle/src/Translator/TranslateAssistantDto.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RZ\Roadiz\RozierBundle\Translator;
+
+final readonly class TranslateAssistantDto
```

Comment:
```suggestion
final readonly class TranslateAssistantInput
```

Why not call it `Input` to be symmetric with your `TranslateAssistantOutput`